app/src/main/java/org/sopt/official/feature/attendance/compose/AttendanceScreen.kt

```diff
@@ -0,0 +1,209 @@
+    ) {
+        Column(
+            modifier = Modifier.fillMaxHeight(),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Spacer(Modifier.height(16.dp))
+            when (state.attendanceDayType) {
+                is AttendanceDayType.AttendanceDay -> {
+                    TodayAttendanceCard(
+                        modifier = Modifier.fillMaxWidth(),
+                        eventDate = state.attendanceDayType.eventDate,
+                        eventLocation = state.attendanceDayType.eventLocation,
+                        eventName = state.attendanceDayType.eventName,
+                        firstAttendance = state.attendanceDayType.firstAttendance,
+                        secondAttendance = state.attendanceDayType.secondAttendance,
+                        finalAttendance = state.attendanceDayType.finalAttendance,
+                    )
+                }
+
+                is AttendanceDayType.Event -> {
+                    TodayNoAttendanceCard(
+                        modifier = Modifier.fillMaxWidth(),
+                        eventDate = state.attendanceDayType.eventDate,
+                        eventLocation = state.attendanceDayType.eventLocation,
+                        eventName = state.attendanceDayType.eventLocation,
+                    )
+                }
+
+                AttendanceDayType.None -> {
+                    TodayNoScheduleCard(
+                        modifier = Modifier.fillMaxWidth()
+                    )
+                }
+            }
+            Spacer(Modifier.height(20.dp))
+            AttendanceHistoryCard(
+                userTitle = state.userTitle,
+                attendanceScore = state.attendanceScore,
+                totalAttendanceResult = state.totalAttendanceResult,
+                attendanceHistoryList = state.attendanceHistoryList,
+                scrollState = scrollState,
+            )
+            Spacer(Modifier.height(36.dp))
+        }
+        AttendanceGradientBox()
+        TextButton(
+            onClick = { /*TODO*/ },
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(bottom = 9.dp),
+            shape = RoundedCornerShape(size = 6.dp),
+            colors = ButtonColors(
+                containerColor = SoptTheme.colors.onSurface10,
+                contentColor = SoptTheme.colors.onSurface950,
+                disabledContainerColor = Black40,
+                disabledContentColor = Gray60,
+            ),
+            contentPadding = PaddingValues(vertical = 16.dp)
+        ) {
+            Text(
+                text = stringResource(R.string.attendance_dialog_button),
+                style = SoptTheme.typography.label18SB,
+            )
+        }
+    }
+}
+
+@Preview
+@Composable
+private fun AttendanceScreenPreview(@PreviewParameter(AttendanceScreenPreviewParameterProvider::class) parameter: AttendanceScreenPreviewParameter) {
```

Comment:
```suggestion
private fun AttendanceScreenPreview(
    @PreviewParameter(AttendanceScreenPreviewParameterProvider::class)
    parameter: AttendanceScreenPreviewParameter
) {
```
이렇게 적어도 괜찮을듯? 패러미터 클래스가 너무기네ㅋㅋ